docs/specifications/deep-links.md

```diff
@@ -0,0 +1,190 @@
+---
+icon: link
+---
+
+# Deep Links
+
+Deep links are special URLs that, when navigated to, open
+applications rather than a website in a browser.
+They are useful for a number of things, however in our case
+we are interested in the user experiences that they enable
+between websites, applications, platforms, etc. by providing more interoperability.
+
+Firefly has its own deep link scheme, exposing (limited) functionality that is required in 
+some type of user flow. A trivial example would be a user is buying native tokens on Soonaverse and must make a payment transaction
+in order to execute the buy order. Clicking on a deep link embedded inside the Soonaverse platform will trigger Firefly to 
+open and auto-fill the transaction data as necessary, making it simple confirm and click job for the user. 
+
+:::caution
+Firefly **will NEVER** automatically execute actions initiated by a deep link; they should **ALWAYS** require manual 
+confirmation on behalf of the user.
+:::
+
+## Scheme
+
+The Firefly deep link scheme can be broken down to the following (simple) syntax:
+
+```
+firefly[-<stage>]://<context>/<operation>[?param=<param>]
+```
+
+The parameters are as follows:
+
+- `stage` - indicates a specific stage of the app to target, options are:
+  - `alpha` - the first available version of Firefly containing brand new features
+  - `beta` - the next available version of Firefly containing new but slightly tested features
+  - `shimmer` - the Firefly Shimmer version, containing new and well-tested features 
+- `context` - the part of Firefly that contains the operation, options are:
+  - `wallet` - managing coins and tokens
+  - `collectibles` - managing NFTs
```

Comment:
does 'nft' make more sense maybe?

Reply:
This is supposed to be a 1:1 mapping with the contexts of the app, so I think `collectibles` would be better.